sdk/ai/azure-ai-inference/samples/sample_chat_completions_with_tracing.py

```diff
@@ -23,6 +23,7 @@
         where `your-deployment-name` is your unique AI Model deployment name, and
         `your-azure-region` is the Azure region where your model is deployed.
     2) AZURE_AI_CHAT_KEY - Your model key (a 32-character string). Keep it secret.
+    3) AZURE_TRACING_GEN_AI_CONTENT_RECORDING_ENABLED - Set to 'true' to enable content recording.
 """
```

Comment:
"to enable content recording."  - When I first saw the name of the env variable I did not understand what "content recording" means. In my azure.ai.projects samples I wrote:
`AZURE_TRACING_GEN_AI_CONTENT_RECORDING_ENABLED=true - Optional. For detailed traces, including chat request and response messages. `
To me this better describes what this "true/false" turns on and off... perhaps we can come up with an even better description.